src/content/app/regulatory-activity-viewer/components/region-activity-section/RegionActivitySection.tsx

```diff
@@ -91,15 +60,31 @@ const RegionActivitySection = (props: Props) => {
         className={regionOverviewStyles.middleColumn}
         ref={imageContainerRef}
       >
-        {currentData && preparedData && width && (
-          <RegionActivitySectionImage
-            width={width}
-            regionOverviewData={currentData}
-            featureTracks={preparedData.featureTracks}
-            start={preparedData.start}
-            end={preparedData.end}
-          />
-        )}
+        <div className={styles.container}>
+          {preparedData && width && (
+            <>
+              <RegionActivitySectionImage
+                width={width}
+                regionOverviewData={preparedData.regionOverviewData}
+                featureTracks={preparedData.featureTracksData}
+                start={preparedData.location.start}
+                end={preparedData.location.end}
+              />
+              {/* A temporary vertical separator component below */}
+              <div style={{ margin: '1rem 0' }} />
```

Comment:
Shouldn't this div be closed rather than be self-enclosing?

Reply:
It's a temporary div; it was easier for me to just add the vertical spacing inline rather than add a css class to something. I expect it to go away when the dust settles. It doesn't have any content, which is why I made it self-closing. Most of self-closing tags aren't valid html, but are valid jsx; react just expands it to `<div style="something"></div>`.